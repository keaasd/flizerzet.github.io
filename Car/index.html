<!DOCTYPE html>
<html lang="ru">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<meta name="description" content="Описание">
	<link rel="shortcut icon" href="/img/favicon.ico" type="image/x-icon">
	<link rel="stylesheet" href="css/style.min.css">
	<title>Sample Car Repair</title>
</head>
<body>
	<div class="wrapper">
		<header class="header">
	<div class='container header__container'>
		<div class="header__top top-header">
			<a href="" class="top-header__logo"><img src="img/logo.svg" alt="" /></a>
			<div class="top-header__info" data-da='menu__body,last,768'>
				<div class="top-header__item item-top-header">
					<img src="img/icons/clock.svg" alt="">
					<div class="item-top-header__info">
						<span>SERVICE HOURS</span>
						<span>Mon - Sat: 7:00 am - 6:00 pm</span>
					</div>
				</div>
				<div class="top-header__item item-top-header">
					<img src="img/icons/phone.svg" alt="">
					<div class="item-top-header__info">
						<span>CALL US</span>
						<a href="tel:1111111111">(555) Car-Repair</a>
					</div>
				</div>
				<button class="_modal-open top-header__btn btn" data-open="appointment"><span>Make an Appointment</span></button>
			</div>
			<div class="hamburger__wrapper">
				<div class="hamburger">
					<span></span>
					<span></span>
					<span></span>
				</div>
			</div>
		</div>
		<div class="header__bottom bottom-header">
			<div class="bottom-header__menu menu">
				<nav class="menu__body">
					<ul class="menu__list">
						<li><a class="menu__link header-dropdown" href="#">Services</a></li>
						<li><a class="menu__link" href="#">Specials</a></li>
						<li><a class="menu__link" href="#">Estimates</a></li>
						<li><a class="menu__link" href="#">Reviews</a></li>
						<li><a class="menu__link header-dropdown" href="#">Shop</a></li>
						<li><a class="menu__link" href="#">About</a></li>
					</ul>
				</nav>
			</div>
			<div class="bottom-header__buttons" data-da="hamburger__wrapper,1,768">
				<button class="bottom-header__button bottom-header-search">
					<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
						<path fill-rule="evenodd" clip-rule="evenodd" d="M14.9649 14.255H15.7549L20.7449 19.255L19.2549 20.745L14.2549 15.755V14.965L13.9849 14.685C12.8449 15.665 11.3649 16.255 9.75488 16.255C6.16488 16.255 3.25488 13.345 3.25488 9.755C3.25488 6.165 6.16488 3.255 9.75488 3.255C13.3449 3.255 16.2549 6.165 16.2549 9.755C16.2549 11.365 15.6649 12.845 14.6849 13.985L14.9649 14.255ZM5.25488 9.755C5.25488 12.245 7.26488 14.255 9.75488 14.255C12.2449 14.255 14.2549 12.245 14.2549 9.755C14.2549 7.265 12.2449 5.255 9.75488 5.255C7.26488 5.255 5.25488 7.265 5.25488 9.755Z"/>
					</svg>						
				</button>
				<button class="bottom-header__button bottom-header-cart">
					<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
						<path fill-rule="evenodd" clip-rule="evenodd" d="M18.2963 11.97C17.9563 12.59 17.2963 13 16.5463 13H9.09634L7.99634 15H19.9963V17H7.99634C6.47634 17 5.51634 15.37 6.24634 14.03L7.59634 11.59L3.99634 4H1.99634V2H5.26634L6.20634 4H21.0063C21.7663 4 22.2463 4.82 21.8763 5.48L18.2963 11.97ZM19.3063 6H7.15634L9.52634 11H16.5463L19.3063 6ZM7.99634 18C6.89634 18 6.00634 18.9 6.00634 20C6.00634 21.1 6.89634 22 7.99634 22C9.09634 22 9.99634 21.1 9.99634 20C9.99634 18.9 9.09634 18 7.99634 18ZM16.0063 20C16.0063 18.9 16.8963 18 17.9963 18C19.0963 18 19.9963 18.9 19.9963 20C19.9963 21.1 19.0963 22 17.9963 22C16.8963 22 16.0063 21.1 16.0063 20Z"/>
					</svg>						
					<span>2</span>
				</button>
			</div>
		</div>
	</div>
</header>
		<main class="page">
			<div class="hero">
				<div class='container hero__container'>
					<h1 class="hero__title">Top notch service is our main auto motive</h1>
					<button class="_modal-open btn hero__btn" data-open="appointment"><span>Make an Appointment</span></button>
				</div>
			</div>
			<div class="who">
				<div class='container who__container'>
					<div class="who__info">
						<div class="who__tag tag">who we are</div>
						<div class="who__title title">Nemo enim ipsam voluptatem quia voluptas in proident</div>
						<div class="who__text text">Ut enim ad minima veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur. Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur.</div>
						<div class="who__img"><img src="img/who-img.png" alt=""></div>
					</div>
				</div>
			</div>
			<div class="promise">
				<div class='container promise__container'>
					<div class="promise__tag tag">OUR PROMISE TO YOU</div>
					<div class="promise__title title">Sed perspiciatis omnis iste natus</div>
					<div class="promise__body">
						<div class="promise__item item-promise">
							<div class="item-promise__img"><img src="img/promise/1.svg" alt=""></div>
							<div class="item-promise__info">
								<div class="item-promise__title">Quality service. Everytime.</div>
								<div class="item-promise__text text">At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium corrupti quos dolores et quas.</div>
							</div>
						</div>
						<div class="promise__item item-promise">
							<div class="item-promise__img"><img src="img/promise/2.svg" alt=""></div>
							<div class="item-promise__info">
								<div class="item-promise__title">Services by certified pros</div>
								<div class="item-promise__text text">Dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt.</div>
							</div>
						</div>
						<div class="promise__item item-promise">
							<div class="item-promise__img"><img src="img/promise/3.svg" alt=""></div>
							<div class="item-promise__info">
								<div class="item-promise__title">Real people. Honest work.</div>
								<div class="item-promise__text text">Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae consequatur illum.</div>
							</div>
						</div>
					</div>
				</div>
			</div>
			<div class="video">
				<div class='container video__container'>
					<div class="video__body">
						<div class="video__title title">Nemo enim ipsam voluptatem quia voluptas in proident</div>
						<button class="_modal-open video__btn" data-open="video">
							<div class="pulse-anim"></div>
						</button>
					</div>
				</div>
			</div>
			<div class="services">
				<div class='container services__container'>
					<div class="services__tag tag">Our Services</div>
					<div class="services__title title">Sed perspiciatis omnis iste natus</div>
					<div class="services__body">
						<div class="services__item item-services">
							<div class="item-services__img"><img src="img/service/1.svg" alt=""></div>
							<div class="item-services__title">Multi Point Inspections</div>
							<div class="item-services__text text">At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium corrupti quos dolores et quas.</div>
						</div>
						<div class="services__item item-services">
							<div class="item-services__img"><img src="img/service/2.svg" alt=""></div>
							<div class="item-services__title">Auto Repair</div>
							<div class="item-services__text text">At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium corrupti quos dolores et quas.</div>
						</div>
						<div class="services__item item-services">
							<div class="item-services__img"><img src="img/service/3.svg" alt=""></div>
							<div class="item-services__title">Auto Maintenance</div>
							<div class="item-services__text text">At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium corrupti quos dolores et quas.</div>
						</div>
						<div class="services__item item-services">
							<div class="item-services__img"><img src="img/service/4.svg" alt=""></div>
							<div class="item-services__title">Tires</div>
							<div class="item-services__text text">At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium corrupti quos dolores et quas.</div>
						</div>
						<div class="services__item item-services">
							<div class="item-services__img"><img src="img/service/5.svg" alt=""></div>
							<div class="item-services__title">Service Warranty</div>
							<div class="item-services__text text">At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium corrupti quos dolores et quas.</div>
						</div>
						<div class="services__item item-services">
							<div class="item-services__img"><img src="img/service/6.svg" alt=""></div>
							<div class="item-services__title">Shop Online</div>
							<div class="item-services__text text">At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium corrupti quos dolores et quas.</div>
						</div>
					</div>
					<a href="#" class="services__btn btn"><span>Learn More</span></a>
				</div>
			</div>
			<div class="appointment">
				<img src="img/bg/appointment-services-bg.png" alt="">
				<div class='container appointment__container'>
					<div class="appointment__form">
						<div class="appointment__form-title title">Make an appointment</div>
						<form action="#" class="form-appointment">
							<div class="form-appointment__inputs">
								<input type="text" class="form-appointment__input" name="name" placeholder="Name" required>
								<input type="email" class="form-appointment__input" name="mail" placeholder="Email" required>
								<input type="tel" class="form-appointment__input" name="phone" placeholder="Phone Number" required>
								<select type="text" class="form-appointment__input form-appointment__select" name="sevice" placeholder="Select Service" required>
									<option value="none">Select Service</option>
									<option value="2">2</option>
									<option value="3">3</option>
								</select>
								<textarea type="text" class="form-appointment__input form-appointment__textarea" name="opinion" placeholder="Tell us about the car needing service(s)"></textarea>
							</div>
							<button class="form-appointment__btn btn" type="submit"><span>Make an Appointment</span></button>
						</form>
					</div>
					<div class="appointment__info">
						<div class="appointment__title">Maxime placeat facere possimus omnis</div>
						<div class="appointment__counter"><span id='counter'>17</span>k</div>
						<div class="appointment__subtitle">customers served (and counting)</div>
					</div>
				</div>
			</div>
			<div class="gallery">
				<div class='container gallery__container'>
					<div class="gallery__body">
						<div class="gallery__tag tag">GALLERY & LIVE STREAMS</div>
						<div class="gallery__title title">Check out our hard workers</div>
						<div class="_tabs gallery__tabs tabs-gallery">
							<div class="tabs__links tabs-gallery__links">
								<div class="_tabs-link _active tabs-gallery__link text">Gallery</div>
								<div class="_tabs-link tabs-gallery__link _online text">Live stream: Bay 1<span></span></div>
								<div class="_tabs-link tabs-gallery__link _online text">Live stream: Bay 2<span></span></div>
								<div class="_tabs-link tabs-gallery__link text">Live stream: Bay 3<span></span></div>
							</div>
							<div class="tabs__contents tabs-gallery__contents">
								<div class="_tabs-content _active tabs-gallery__content"><img src="img/stream/1.png" alt=""></div>
								<div class="_tabs-content tabs-gallery__content"><img src="img/stream/2.png" alt=""></div>
								<div class="_tabs-content tabs-gallery__content"><img src="img/stream/1.png" alt=""></div>
								<div class="_tabs-content tabs-gallery__content"><img src="img/stream/2.png" alt=""></div>							
							</div>
						</div>
					</div>
				</div>
			</div>
			<div class="customers">
				<div class='container customers__container'>
					<div class="customers__info">
						<div class="customers__tag tag">Our CUSTOMERS</div>
						<div class="customers__title title">Illum qui dolorem eum quo voluptas ut distinctio</div>
					</div>
					<div class="customers__body">
						<div class="customers__card card-customers">
							<div class="card-customers__text text">“Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore ex magna aliqua. Ut enim illum ad minim veniam, quis ea exercitation ullamco cillum ut enim dolore.”</div>
							<div class="card-customers__user">
								<div class="card-customers__avatar"><img src="img/avatars/1.jpeg" alt=""></div>
								<div class="card-customers__name">Cullen Bohannon</div>
							</div>
						</div>
						<div class="customers__card card-customers">
							<div class="card-customers__text text">“Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi ipsam voluptatem aspernatur.”</div>
							<div class="card-customers__user">
								<div class="card-customers__avatar"><img src="img/avatars/2.jpeg" alt=""></div>
								<div class="card-customers__name">Jennifer Thomas</div>
							</div>
						</div>
					</div>
				</div>
			</div>
			<div class="impact">
				<div class='container impact__container'>
					<img src="img/impact/imoact-img.png" alt="" class="impact__img">
					<div class="impact__body">
						<div class="impact__tag tag">OUR IMPACT</div>
						<div class="impact__title title">Nemo enim ipsam voluptatem quia voluptas</div>
						<div class="impact__items">
							<div class="impact__item item-impact">
								<div class="item-impact__img"><img src="img/impact/1.svg" alt=""></div>
								<div class="item-impact__info">
									<div class="item-impact__title">16k+</div>
									<div class="item-impact__subtitle">Oil changes</div>
								</div>
							</div>
							<div class="impact__item item-impact">
								<div class="item-impact__img"><img src="img/impact/2.svg" alt=""></div>
								<div class="item-impact__info">
									<div class="item-impact__title">13k+</div>
									<div class="item-impact__subtitle">Transmission repairs</div>
								</div>
							</div>
							<div class="impact__item item-impact">
								<div class="item-impact__img"><img src="img/impact/3.svg" alt=""></div>
								<div class="item-impact__info">
									<div class="item-impact__title">15k+</div>
									<div class="item-impact__subtitle">Alignments made</div>
								</div>
							</div>
							<div class="impact__item item-impact">
								<div class="item-impact__img"><img src="img/impact/4.svg" alt=""></div>
								<div class="item-impact__info">
									<div class="item-impact__title">11,500+</div>
									<div class="item-impact__subtitle">Engines repaired</div>
								</div>
							</div>
						</div>
					</div>
				</div>
			</div>
			<div class="prefooter">
				<div class='container prefooter__container'>
					<div class="prefooter__text">Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua</div>
					<button class="_modal-open btn prefooter__btn" data-open="appointment"><span>Make an Appointment</span></button>
				</div>
			</div>
		</main>
		<footer class="footer">
	<div class='container footer__container'>
		<div class="footer__header header-footer">
			<a href="#" class="header-footer__logo" data-da='body-footer__col_1,first,768'><img src="img/logo-b.svg" alt=""></a>
			<div class="header-footer__title" data-da='body-footer__col_2,first,768'>EXPLORE LINKS</div>
			<div class="header-footer__title" data-da='body-footer__col_3,first,768'>CONTACT</div>
		</div>
		<div class="footer__body body-footer">
			<div class="body-footer__col body-footer__col_1">
				<div class="body-footer__text text">This is a blurb about the company sed do eiusmod tempor incididunt ut labore et dolore ex magna aliqua.</div>
				<div class="body-footer__socials">
					<a href="#" class="body-footer__social">
						<img src="img/icons/socials/fb.svg" alt="">
					</a>
					<a href="#" class="body-footer__social">
						<img src="img/icons/socials/twitter.svg" alt="">
					</a>
					<a href="#" class="body-footer__social">
						<img src="img/icons/socials/inst.svg" alt="">
					</a>
					<a href="#" class="body-footer__social">
						<img src="img/icons/socials/yt.svg" alt="">
					</a>
				</div>
			</div>
			<div class="body-footer__col body-footer__col_2">
				<div class="body-footer__body">
					<ul class="body-footer__list">
						<li><a href="#" class="body-footer__link">About</a></li>
						<li><a href="#" class="body-footer__link">Services</a></li>
						<li><a href="#" class="body-footer__link">Specials</a></li>
						<li><a href="#" class="body-footer__link">Estimates</a></li>
					</ul>
					<ul class="body-footer__list">
						<li><a href="#" class="body-footer__link">Reviews</a></li>
						<li><a href="#" class="body-footer__link">Shop</a></li>
						<li><a href="#" class="body-footer__link">Careers</a></li>
						<li><a href="#" class="body-footer__link">FAQs</a></li>
					</ul>
				</div>
			</div>
			<div class="body-footer__col body-footer__col_3">
				<div class="body-footer__info body-footer__info_place">321 Pikes Place Parkway Seattle, WA 54321</div>
				<div class="body-footer__info body-footer__info_smartphone"><a href="tel:5557654321">(555) 765-4321</a></div>
				<div class="body-footer__info body-footer__info_email"><a href="mailto:info@company.com">Info@company.com</a></div>
				<div class="body-footer__info body-footer__info_timer">Mon - Sat:  7:000 am - 6:00 pm</div>
			</div>
		</div>
	</div>
	<div class="footer__tagline tagline">
		<div class='container tagline__container'>
			<div class="tagline__text">Tagline lorem ipsum dolor sit amet, consectetur adipiscing eiusmod</div>
			<div class="tagline__copy">
				<div class="tagline__date">© 2018 Company.</div>
				<div class="tagline__links">
					<div class="tagline__link">
						<a href="#">Privacy Policy</a>
					</div>
					<div class="tagline__link">
						<a href="#">Terms & Conditions</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</footer>
	</div>
	<div class="_modal appointment__modal _hide" data-modal="appointment">
		<div class="_modal-bg">
			<div class="_modal-content">
				<div class="_modal-close">&times;</div>
				<div class="appointment__form">
					<div class="appointment__form-title title">Make an appointment</div>
					<form action="#" class="form-appointment">
						<div class="form-appointment__inputs">
							<input type="text" class="form-appointment__input" name="name" placeholder="Name" required>
							<input type="email" class="form-appointment__input" name="mail" placeholder="Email" required>
							<input type="tel" class="form-appointment__input" name="phone" placeholder="Phone Number" required>
							<select type="text" class="form-appointment__input form-appointment__select" name="sevice" placeholder="Select Service" required>
								<option value="none">Select Service</option>
								<option value="2">2</option>
								<option value="3">3</option>
							</select>
							<textarea type="text" class="form-appointment__input form-appointment__textarea" name="opinion" placeholder="Tell us about the car needing service(s)"></textarea>
						</div>
						<button class="form-appointment__btn btn" type="submit"><span>Make an Appointment</span></button>
					</form>
				</div>
			</div>
		</div>
	</div>
	<div class="_modal video__modal _hide" data-modal="video">
		<div class="_modal-bg">
			<div class="_modal-content">
				<div class="_modal-close">&times;</div>
				<iframe src="https://www.youtube.com/embed/dQw4w9WgXcQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
			</div>
		</div>
	</div>
	<script src="js/libs.js" defer></script>
	<script src="js/script.min.js" defer></script>
</body>
</html>