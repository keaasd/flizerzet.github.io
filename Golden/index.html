<!DOCTYPE html><html lang="en">	<head>		<meta charset="UTF-8">		<meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">		<meta http-equiv="X-UA-Compatible" content="ie=edge">		<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600&display=swap" rel="stylesheet">		<link href="https://fonts.googleapis.com/css2?family=Roboto+Slab&display=swap" rel="stylesheet">		<link rel="stylesheet" href="css/style.min.css">		<title>Golden</title>	</head>	<body>		<div class="wrapper">			<header class="header">				<div class='container'>					<div class="header__nav">						<div class="header__logo">							<a href="#"><picture><source srcset="img/logo.webp" type="image/webp"><img src="img/logo.png" alt="" /></picture></a>						</div>						<ul class="header__links">							<li class="header__link"><a href="#">Home</a></li>							<li class="header__link"><a href="#services">Services</a></li>							<li class="header__link"><a href="#portfolio">Portfolio</a></li>							<li class="header__link"><a href="#about">About</a></li>							<li class="header__link"><a href="#callback">Contact</a></li>						</ul>						<div class="hamburger">							<span></span>							<span></span>							<span></span>						</div>					</div>					<div class="header__suptitle">Welcome To Our Studio!</div>					<div class="header__title">Itâ€™s nice to Meet you</div>					<a class="btn" href="#">Tell Me More</a>				</div> 			</header>			<div id="services" class="services">				<div class='container'>					<div class="title">Services</div>					<div class="subtitle">Proin iaculis purus consequat sem cure.</div>					<div class="services__row">						<div class="services__col">							<div class="services__icon">								<picture><source srcset="img/icons/basket.svg" type="image/webp"><img src="img/icons/basket.svg" alt="" /></picture>							</div>							<div class="services__title">E-Commerce</div>							<div class="services__text">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</div>						</div>						<div class="services__col">							<div class="services__icon">								<picture><source srcset="img/icons/laptop.svg" type="image/webp"><img src="img/icons/laptop.svg" alt="" /></picture>							</div>							<div class="services__title">Responsive Web</div>							<div class="services__text">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</div>						</div>						<div class="services__col">							<div class="services__icon">								<picture><source srcset="img/icons/lock.svg" type="image/webp"><img src="img/icons/lock.svg" alt="" /></picture>							</div>							<div class="services__title">Web Security</div>							<div class="services__text">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</div>						</div>					</div>				</div>			</div>			<div id="portfolio" class="portfolio">				<div class='container'>					<div class="title">Our portfolio</div>					<div class="subtitle">Proin iaculis purus consequat sem cure.</div>					<div class="portfolio__row">						<a href="#" class="portfolio__item">							<div class="portfolio__img">								<picture><source srcset="img/portf_1.webp" type="image/webp"><img src="img/portf_1.png" alt="" /></picture>							</div>							<div class="portfolio__info">								<div class="portfolio__title">Ebony & Ivory</div>								<div class="portfolio__descr">Branding</div>							</div>						</a>						<a href="#" class="portfolio__item">							<div class="portfolio__img">								<picture><source srcset="img/portf_2.webp" type="image/webp"><img src="img/portf_2.png" alt="" /></picture>							</div>							<div class="portfolio__info">								<div class="portfolio__title">Smart Stationary</div>								<div class="portfolio__descr">Print Design</div>							</div>						</a>						<a href="#" class="portfolio__item">							<div class="portfolio__img">								<picture><source srcset="img/portf_3.webp" type="image/webp"><img src="img/portf_3.png" alt="" /></picture>							</div>							<div class="portfolio__info">								<div class="portfolio__title">Clever Poster</div>								<div class="portfolio__descr">Print Design</div>							</div>						</a>						<a href="#" class="portfolio__item">							<div class="portfolio__img">								<picture><source srcset="img/portf_4.webp" type="image/webp"><img src="img/portf_4.png" alt="" /></picture>							</div>							<div class="portfolio__info">								<div class="portfolio__title">Vinyl Record</div>								<div class="portfolio__descr">Product Mock-Up</div>							</div>						</a>						<a href="#" class="portfolio__item">							<div class="portfolio__img">								<picture><source srcset="img/portf_5.webp" type="image/webp"><img src="img/portf_5.png" alt="" /></picture>							</div>							<div class="portfolio__info">								<div class="portfolio__title">Treehouse Template</div>								<div class="portfolio__descr">Web Design</div>							</div>						</a>						<a href="#" class="portfolio__item">							<div class="portfolio__img">								<picture><source srcset="img/portf_6.webp" type="image/webp"><img src="img/portf_6.png" alt="" /></picture>							</div>							<div class="portfolio__info">								<div class="portfolio__title">Burned Logo</div>								<div class="portfolio__descr">Branding</div>							</div>						</a>					</div>				</div>			</div>			<div id="about" class="about">				<div class='container'>					<div class="title">About Us</div>					<div class="subtitle">Proin iaculis purus consequat sem cure.</div>					<div class="about__col">						<div class="about__item">							<div class="about__info about__info_l">								<div class="about__title"><span>July 2010</span><br>Our Humble Beginnings</div>								<div class="about__text">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</div>							</div>							<div class="about__img">								<picture><source srcset="img/about_1.webp" type="image/webp"><img src="img/about_1.png" alt="" /></picture>							</div>							<div class="none"></div>						</div>						<div class="about__item">							<div class="none"></div>							<div class="about__img">								<picture><source srcset="img/about_2.webp" type="image/webp"><img src="img/about_2.png" alt="" /></picture>							</div>							<div class="about__info">								<div class="about__title"><span>January 2011</span><br>Facing Startup Battles</div>								<div class="about__text">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</div>							</div>						</div>						<div class="about__item">							<div class="about__info about__info_l">								<div class="about__title"><span>December 2012</span><br>Enter The Dark Days</div>								<div class="about__text">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni.</div>							</div>							<div class="about__img">								<picture><source srcset="img/about_3.webp" type="image/webp"><img src="img/about_3.png" alt="" /></picture>							</div>							<div class="none"></div>						</div>						<div class="about__item">							<div class="none"></div>							<div class="about__img">								<picture><source srcset="img/about_2.webp" type="image/webp"><img src="img/about_2.png" alt="" /></picture>							</div>							<div class="about__info">								<div class="about__title"><span>february 2014</span><br>Our Triumph</div>								<div class="about__text">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean.</div>							</div>						</div>						<div class="about__item">							<div></div>							<a href="#" class="about__img_t">								Our<br>Story<br>Continues<br>...							</a>							<div class="none"></div>						</div>					</div>				</div>			</div>			<div id="team" class="team">				<div class='container'>					<div class="title">our amazing team</div>					<div class="subtitle">Proin iaculis purus consequat sem cure.</div>					<div class="team__row">						<div class="team__col">							<div class="team__item">								<div class="team__avatar"></div>								<div class="team__info">									<div class="team__name">Kimberly Thompson</div>									<div class="team__prof">Marketer</div>									<div class="team__social">										<a href="#" class="team__icon">											<picture><source srcset="img/icons/Twitter.webp" type="image/webp"><img src="img/icons/Twitter.png" alt="" /></picture>										</a>										<a href="#" class="team__icon">											<picture><source srcset="img/icons/Facebook.webp" type="image/webp"><img src="img/icons/Facebook.png" alt="" /></picture>										</a>										<a href="#" class="team__icon">											<picture><source srcset="img/icons/Pinterest.webp" type="image/webp"><img src="img/icons/Pinterest.png" alt="" /></picture>										</a>									</div>								</div>							</div>						</div>						<div class="team__col">							<div class="team__item">								<div class="team__avatar"></div>								<div class="team__info">									<div class="team__name">Rico Massimo</div>									<div class="team__prof">Coder</div>									<div class="team__social">										<a href="#" class="team__icon">											<picture><source srcset="img/icons/Twitter.webp" type="image/webp"><img src="img/icons/Twitter.png" alt="" /></picture>										</a>										<a href="#" class="team__icon">											<picture><source srcset="img/icons/Facebook.webp" type="image/webp"><img src="img/icons/Facebook.png" alt="" /></picture>										</a>										<a href="#" class="team__icon">											<picture><source srcset="img/icons/Google.webp" type="image/webp"><img src="img/icons/Google.png" alt="" /></picture>										</a>									</div>								</div>							</div>						</div>						<div class="team__col">							<div class="team__item">								<div class="team__avatar"></div>								<div class="team__info">									<div class="team__name">Uku Mason</div>									<div class="team__prof">Graphic Designer</div>									<div class="team__social">										<a href="#" class="team__icon">											<picture><source srcset="img/icons/Facebook.webp" type="image/webp"><img src="img/icons/Facebook.png" alt="" /></picture>										</a>										<a href="#" class="team__icon">											<picture><source srcset="img/icons/Pinterest.webp" type="image/webp"><img src="img/icons/Pinterest.png" alt="" /></picture>										</a>									</div>								</div>							</div>						</div>					</div>					<div class="team__ps">Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt proin iaculis.</div>				</div>			</div>			<div class="brands">				<div class='container'>					<div class="brands__row">						<div class="brands__item">							<picture><source srcset="img/Envato.webp" type="image/webp"><img src="img/Envato.png" alt="" /></picture>						</div>						<div class="brands__item">							<picture><source srcset="img/WordPress.webp" type="image/webp"><img src="img/WordPress.png" alt="" /></picture>						</div>						<div class="brands__item">							<picture><source srcset="img/Tuts.webp" type="image/webp"><img src="img/Tuts.png" alt="" /></picture>						</div>						<div class="brands__item">							<picture><source srcset="img/Microlancer.webp" type="image/webp"><img src="img/Microlancer.png" alt="" /></picture>						</div>					</div>				</div>			</div>			<div id="callback" class="callback">				<div class='container'>					<div class="callback__title title title_w">Contact us</div>					<div class="subtitle">Proin iaculis purus consequat sem cure.</div>					<form class="callback__form" action="#" enctype="multipart/form-data">						<div class="callback__col">							<div class="callback__item callback__item_i">								<input required class="callback__input" type="text" placeholder="your Name *">								<input required class="callback__input" type="email" placeholder="your e-mail *">								<input class="callback__input" type="text" placeholder="subject">							</div>							<div class="callback__item">								<textarea required class="callback__textarea" name="message" placeholder="Your message *"></textarea>							</div>						</div>						<button type="submit" class="btn">Send Message</button>					</form>				</div>			</div>			<footer class="footer">				<div class='container'>					<div class="footer__links">						<a href="#" class="footer__item">							<picture><source srcset="img/icons/Twitter.webp" type="image/webp"><img src="img/icons/Twitter.png" alt="" /></picture>						</a>						<a href="#" class="footer__item">							<picture><source srcset="img/icons/Facebook.webp" type="image/webp"><img src="img/icons/Facebook.png" alt="" /></picture>						</a>						<a href="#" class="footer__item">							<picture><source srcset="img/icons/Pinterest.webp" type="image/webp"><img src="img/icons/Pinterest.png" alt="" /></picture>						</a>						<a href="#" class="footer__item">							<picture><source srcset="img/icons/Google.webp" type="image/webp"><img src="img/icons/Google.png" alt="" /></picture>						</a>					</div>					<div class="footer__text">Â© Copyright 2014 FreebiesXpress.com</div>				</div>			</footer>		</div>		<script src="js/script.js"></script>	</body></html>